ecs-init/config/config_unspecified.go

```diff
@@ -0,0 +1,18 @@
+// +build !suse,!ubuntu
+
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+// http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package config
+
+const cgroupMountpoint = "/cgroup"
```

Comment:
I don't know on what other distro's this is true. 
I'm only aware of this on `Amazon Linux`. 

`CoreOS` for example uses `/sys/fs/cgroup`

Reply:
I'm following the convention elsewhere in the package, where we default to Amazon Linux and have overrides in place for Suse and Ubuntu. 

We don't package ecs-init for CoreOS .

Reply:
I'd think that `/cgroup` is rather specific only for `Amazon Linux` as most traditional distro's use `/sys/fs/cgroup`. 

I'd rather have a specific override for `Amazon Linux` and let the other unspecified ones default to `/sys/fs/cgroup`. 

I was not implying that we package things for `CoreOS`, just used it as an example as they have come back and reported issues with the `amazon-ecs-agent`.

Reply:
Right now we default to assuming Amazon Linux and treat other distributions as special-cases.  I think it's reasonable to do that for this right now.

Reply:
It is specific for Amazon Linux -- but that's our default. Our build tools assume its for Amazon Linux unless otherwise specified via platform specific target.

Reply:
Thanks a lot for all the explanations, the file name `config_unspecified.go` and the lack of comments made it confusing for me.